extensions/amp-bind/0.1/test/test-bind-impl.js

```diff
@@ -194,6 +194,17 @@ describes.realWin('Bind', {
     });
   });
 
+  it('should verify class bindings in dev mode', () => {
+    window.AMP_MODE = {development: true};
+    createElementWithBinding(`[class]="'foo'" class="foo"`); // No error.
+    createElementWithBinding(`[class]="'bar'"`); // Error.
```

Comment:
nit:  Please make the reason for the error more explicit in the test. Some ways to do this could be   adding a comment that the values need to match or changing the second element to have an explicitly incorrect value for `class`